scripts/check_nodejs_import_compatibility.js

```diff
@@ -0,0 +1,26 @@
+import fs from "fs";
+import RxPlayer from "../minimal/index.js";
+import * as Features from "../features/index.js";
+import * as Tools from "../experimental/tools/index.js";
+
+let version = fs.readFileSync("./VERSION").toString();
+const indexOfFirstLineBreack = version.indexOf("\n");
+if (indexOfFirstLineBreack >= 0) {
+  version = version.substring(0, indexOfFirstLineBreack);
+}
+
+if (RxPlayer.version !== version) {
+  throw new Error(
+    "Wrong RxPlayer version: " + "\n" +
+    `Got "${RxPlayer.version}" but expected "${version}"`);
+}
+if (typeof Features !== "object" || Features === null) {
```

Comment:
`null` or `undefined` ?

Reply:
We already check that `typeof Features !== "object"`